Common/src/CConfig.cpp

```diff
@@ -2911,6 +2902,16 @@ void CConfig::SetConfig_Parsing(istream& config_buffer){
           /*else if (!option_name.compare("CONV_CRITERIA"))
             newString.append(string("CONV_CRITERIA is deprecated. SU2 will choose the criteria automatically based on the CONV_FIELD.\n") +
                              string("RESIDUAL for any RMS_* BGS_* value. CAUCHY for coefficients like DRAG etc.\n\n"));*/
+          if (!option_name.compare("THERMAL_DIFFUSIVITY"))
+            newString.append("THERMAL_DIFFUSIVITY is deprecated. See the INC_ENERGY_EQUATION options instead.\n\n");
+          if (!option_name.compare("THERMAL_DIFFUSIVITY_SOLID"))
+            newString.append("THERMAL_DIFFUSIVITY_SOLID is deprecated. Set KT_CONSTANT, MATERIAL_DENSITY and SPECIFIC_HEAT_CP instead.\n\n");
+          if (!option_name.compare("SOLID_THERMAL_CONDUCTIVITY"))
+            newString.append("SOLID_THERMAL_CONDUCTIVITY is deprecated. Use KT_CONSTANT instead.\n\n");
+          if (!option_name.compare("SOLID_DENSITY"))
+            newString.append("SOLID_DENSITY is deprecated. Use MATERIAL_DENSITY instead.\n\n");
+          if (!option_name.compare("SOLID_TEMPERATURE_INIT"))
+            newString.append("SOLID_TEMPERATURE_INIT is deprecated. Use FREESTREAM_TEMPERATURE instead.\n\n");
```

Comment:
We will probably call the next version 7.2.0, and so it is fine by me to make changes to options.
However, let's give this some thought based on what would make sense for the scalar solver / flamelets and so on.
We don't need to get the code perfect but "interface" stability is important.

Reply:
Unfortunately I don't have the clear overview (yet, fingers crossed) and what can/needs to be generalized. 

The only thing I would like to change rn would be `KT_CONSTANT` -> `THERMAL_CONDUCTIVITY_CONSTANT` as I feel that is much more expressive.

Reply:
+1 on the expressiveness consideration, it also makes the "did you mean" mechanism work better.